added html header and mobile-minu, teacher section

TetianaZelena opened the pull request:
added base style for body, js file mobail menu

TetianaZelena committed:
added html header and mobile-minu, teacher section

added base style for body, js file mobail menu

pelia-pelia commented on src/partials/header.html:
```diff
@@ -0,0 +1,37 @@
+   <div class="header__navigation container">
+     <nav class="header__menu">
+       <!--Logo -->
+       <div class="header__logo"> <a href="./index.html" class="header__logo--link">Hell<svg class="heder__icon--logo"
```
пропустила букву А в header__icon

TetianaZelena replied:
done

TetianaZelena committed:
fixet bugs

fixed "a" in the name class header__icon, fixed name section from teacher to about

pelia-pelia committed:
Merge branch 'main' into feature/headerhtml

pelia-pelia committed:
fix
